src/GraphQL/Resolvers/AsyncFieldResolver.cs

```diff
@@ -33,7 +33,7 @@ object IFieldResolver.Resolve(IResolveFieldContext context)
 
     /// <summary>
     /// When resolving a field, this implementation calls a predefined <see cref="Func{T, TResult}"/> and returns the result.
-    /// The returned value must be of an <see cref="Task{TResult}"/> type.<br/>
```

Comment:
All these `<br/>` are necessary to create the blank line between paragraphs. Please do not remove them.

Reply:
The text was not so big to break it into paragraphs. Is it important for you?

Reply:
I believe it looked very clean, for the same reason it looks clean in the code.  Visual Studio only puts like half a line of space between paragraphs anyway. Any text that is not separated into paragraphs should not have any `<br/>` so that it is a single paragraph.

Reply:
done with doubled br on separate line
```
<br/><br/>
```